internal/policy/policy_test.go

```diff
@@ -530,3 +532,45 @@ func TestStateHasFileRule(t *testing.T) {
 		assert.False(t, hasFileRule)
 	})
 }
+
+func TestApply(t *testing.T) {
```

Comment:
I'd like to see more cases, split up with `t.Run` that also inspects the policy and policy-staging refs post apply.

Reply:
I have inspected the policy and policy-staging refs, but I do not know how to create an invalid state. How should I do so?